### ISSUE TITLE
Status not staying?

### ISSUE THREAD

**Issue by Paradoof** [AdrienC/discord-deezer-chrome#1]
Heyo!
Nice little tool you put there together - always wanted to display the music I was playing via. Deezer for quite some time.
I have one little problem with it though, the status "Listening to ..."-status seems to disappear too quickly when I checked it with my dummy account. The status seems to be displayed for roughly two minutes before my Status is changed to a blank one so it basicly displays my current title for two minutes, roughly one to four minutes of nothing based on song length and updates my status again when a new song plays and repeats this.
Maybe increasing the base display timer to something high and then it getting overwritten by the next song could help perhaps?

**Comment by AdrienC** [comment]
Hey,
I can't reproduce it, and there's no display timer.
Maybe your dummy account replaces your main account for the status updates ?
- Go to the extension options : chrome-extension://gpmnalhimebpjhilkifcaadejhmogaok/options.html
- Set 'Discord accounts used' to 'All accounts detected, and add the accounts that will be detected' (Otherwise the status updates are made only on the last account you logged in)
- Then, the extension will update the statuses for both of your accounts, and you may see your status updates from both of your accounts.

Tell me if it solves the issue.

**Comment by Paradoof** [comment]
Heya
Firstly thanks for taking your time to help me try to shoot down my problem!
I set it now to 'All accounts detected' and got it to detect my second account with ease - both are now showing my current played title... for two minutes. 😞 
I really don't know why it does this weird thing to me. I even have the game activity thing already disabled (..is playing [blank]) because at first I thought it could interfere with the displayed status but that did nothing.

**Comment by AdrienC** [comment]
I can try to check your logs:
- Enable 'Logging in console' in options : [chrome-extension://gpmnalhimebpjhilkifcaadejhmogaok/options.html](url)
- Enable 'developer mode' at the top of chrome extensions page : [chrome://extensions/](url)
- Then, on the extensions page, in 'Deezer Discord Integration', click 'check view: background'
- Wait for your status to be removed while listening to 'Suicidal Tendencies - Go Skate'
- Copy-paste your log here, and **REMOVE YOUR TOKENS** informations (it looks like this : MuDYw3r7Ge3h8J9Sl2gLkKhq.NdSlFW.IjnHBj6008K38J7n23b7jykjbJH).

**Comment by Paradoof** [comment]
Here you go: [https://pastebin.com/7ECAcBQn](url)
Changed tokens of both accounts to either "9876543212345678998765432123456789" or "ABCDEFGHIJKLMNOPQRSTUBWXYZ00000000".
One thing I noticed was a mid-song 'closed connection' thing but I guess you can read more out of this than I can. 😅

**Comment by AdrienC** [comment]
I don't see anything strange. The closed connection is not a problem.
Do you have any other self bot installed?
Or anything that could change your status?
Are you using the web version of Discord or the desktop app?

**Comment by Paradoof** [comment]
1: Nope
2: Not that I know of. Only thing that Discord does is putting me to 'Idle' after a certain time but that's nothing I could turn off or modify iirc.
3: I tried it on my main account via Desktop and web and with my dummy account on web and mobile. 

I also used a web application that lets me navigate through deezer via the multimedia buttons but turning that off didn't do anything either. Other Chrome extensions I use are uBlock, RES (Reddit Enhancement Suite) and Enhanced Steam but I can't see them being a issue.

If it's nothing that could be fixed from your end then I have to wait for some official Deezer integration by Discord but that will probably take a long while. 😅
